src/Hystrix.Dotnet/project.json

```diff
@@ -0,0 +1,64 @@
+{
+  "version": "1.0.0-*",
+  "description": "A combination of circuit breaker and timeout. The .net version of the open source Hystrix library built by Netflix.",
+
+  "packOptions": {
+    "projectUrl": "https://github.com/Travix-International/Hystrix.Dotnet",
+    "licenseUrl": "https://raw.githubusercontent.com/Travix-International/Hystrix.Dotnet/master/LICENSE",
+    "summary": ".net version of Netflix' Hystrix library.",
+    "requireLicenseAcceptance": false,
+    "owners": [ "Travix International" ],
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/Travix-International/Hystrix.Dotnet.git"
+    }
+  },
+
+  "authors": [ "Travix International" ],
+  "tooling": {
+    "defaultNamespace": "Hystrix.Dotnet"
+  },
+  
+  "buildOptions": {
+	  "warningsAsErrors": true,
+    "nowarn": [ "0618" ]
+  },
+
+  "dependencies": {
+    "JitterMagic": "1.1.2",
+    "Newtonsoft.Json": "9.0.1"
+  },  
+
+  "frameworks": {
+    "netstandard1.5": {
+      "buildOptions": {
+        "define": [ "COREFX" ]
+      },
+      "dependencies": {
+        "NETStandard.Library": "1.6.0",
+        "System.Threading.Thread": "4.0.0",
+        "System.Net.Http": "4.1.0",
+        "System.Diagnostics.Contracts": "4.0.1",
+        "System.Reflection": "4.1.0"
+      }
+    },
+    "netcoreapp1.0": {
```

Comment:
It should, but it didn't so far. Have to retry though to see if I can get it to work. Otherwise we're in a lot of trouble with all the other nuget packages that only target netstandard.